core/web/evm_transactions_controller.go

```diff
@@ -47,3 +60,131 @@ func (tc *TransactionsController) Show(c *gin.Context) {
 
 	jsonAPIResponse(c, presenters.NewEthTxResourceFromAttempt(*ethTxAttempt), "transaction")
 }
+
+type EvmTransactionController struct {
+	Enabled     bool
+	Logger      logger.SugaredLogger
+	TxmStorage  txmgr.EvmTxStore
+	AuditLogger audit.AuditLogger
+	Chains      evm.LegacyChainContainer
+	KeyStore    keystore.Eth
+}
+
+func NewEVMTransactionController(app chainlink.Application) *EvmTransactionController {
+	return &EvmTransactionController{
+		Enabled:     app.GetConfig().TxmAsService().Enabled(),
+		TxmStorage:  app.TxmStorageService(),
+		Logger:      app.GetLogger(),
+		AuditLogger: app.GetAuditLogger(),
+		Chains:      app.GetRelayers().LegacyEVMChains(),
+		KeyStore:    app.GetKeyStore().Eth(),
+	}
+}
+
+// Create signs and sends transaction from specified address. If address is not provided uses one of enabled keys for
+// specified chain.
+func (tc *EvmTransactionController) Create(c *gin.Context) {
+	if !tc.Enabled {
+		jsonAPIError(c, http.StatusUnprocessableEntity, errors.New("transactions creation disabled. To enable set TxmAsService.Enabled=true"))
+		return
+	}
+	var tx models.CreateEVMTransactionRequest
+	if err := c.ShouldBindJSON(&tx); err != nil {
+		jsonAPIError(c, http.StatusBadRequest, err)
+		return
+	}
+
+	if tx.IdempotencyKey == "" {
+		jsonAPIError(c, http.StatusBadRequest, errors.New("idempotencyKey must be set"))
```

Comment:
Actually this is optional in the txm service.
Did you mean here that any caller using the rest api should set it, just in case their connection gets lost, and we create the Tx anyways?
So it will be optional in the service, but required for rest clients?